helpers/env-variables.ts

```diff
@@ -242,6 +244,21 @@ const getModelEnvs = (modelConfig: ModelConfig): EnvVar[] => {
           },
         ]
       : []),
+    ...(modelConfig.provider === "t-systems"
+      ? [
+          {
+            name: "T_SYSTEMS_LLMHUB_BASE_URL",
+            description:
+              "The base URL for the T-Systems AI Foundation Model API. Eg: http://localhost:11434",
+            value: TSYSTEMS_LLMHUB_API_URL,
+          },
+          {
+            name: "T_SYSTEMS_LLMHUB_API_KEY",
```

Comment:
@mohdamir PR is good to go - great job. Just one thing. Sometimes you're using `T_SYSTEMS_LLMHUB_API_KEY` and sometimes `LLMHUB_API_KEY` - the API key should be consistent

Reply:
@mohdamir this still seems to be an issue

Reply:
@marcusschiesser  Thanks for highlighting this. I have fixed and pushed changes. Please review and approve the same. Thanks